fidesctl/src/fidesapi/main.py

```diff
@@ -67,10 +68,29 @@ async def log_request(request: Request, call_next: Callable) -> Response:
     return response
 
 
-@app.get("/health", tags=["Health"])
+@app.get(
+    "/health",
+    response_model=Dict[str, str],
+    responses={
+        status.HTTP_200_OK: {
+            "content": {
+                "application/json": {
+                    "example": {
+                        "status": "healthy",
+                        "version": "1.0.0",
```

Comment:
Alternatively, we can do:
```suggestion
                        "version": str(fidesctl.__version__),
```
which would dynamically populate the auto-docs with the current version. This was giving me some ugly output when running things locally, so I went with a static example just to keep it clean.